<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvexHullWindowClass</class>
 <widget class="QDialog" name="ConvexHullWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>278</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>278</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>包络矩形转面</string>
  </property>
  <widget class="QPushButton" name="cancelpushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startpushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>261</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>输入图层：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>输出要素：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
